coder_upgrade.module

```diff
@@ -162,16 +167,12 @@ function coder_upgrade_build_config_function($node, $reader) {
   $function->parameters = new PGPList();
 
   $strings[] = '$prefixes[\'' . $_coder_upgrade_module_name . '.settings\'] = array(';
-  $strings[] = "  'label' => t('Module name settings'),";
+  $strings[] = "  'label' => t('<Module name> settings'),";
```

Comment:
Is it not possible to get the actual module name and replace there?

Reply:
I didn't see where it would be found. Pointer would be welcome! I just added the angle brackets to make it clear that this was something that needed to be manually replaced (that wasn't clear to me the first few times I coder_upgraded a module).

Reply:
Found it! Latest version of the PR puts the actual human module name in the `hook_config_info()` implementation.